nsxt/policy_utils_test.go

```diff
@@ -0,0 +1,101 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type policyPathTest struct {
+	path    string
+	parents []string
+}
+
+func TestParseStandardPolicyPath(t *testing.T) {
+
+	testData := []policyPathTest{
```

Comment:
nit: add negative test with invalid path and verify code fails as expected.

Reply:
There is a negative test below